src/librbd/CMakeLists.txt

```diff
@@ -9,9 +9,18 @@ set(librbd_types_srcs
   watcher/Types.cc
   WatchNotifyTypes.cc)
 
+if(WITH_RBD_RWL OR WITH_RBD_SSD_CACHE)
+ list(APPEND librbd_types_srcs cache/pwl/Types.cc)
+endif()
+
 add_library(rbd_types STATIC
   ${librbd_types_srcs})
 
+if (WITH_RBD_RWL AND WITH_RBD_SSD_CACHE)
```

Comment:
Nit: no need for `pmem` if just SSD, right?

Reply:
Yes, as no encoding/decoding for pmem.

Reply:
... so should it be `if (WITH_RBD_SSD_CACHE)`?

Reply:
Ack -- I understand now. Thx.